en/docs/references/error-catalog.md

```diff
@@ -1627,3 +1627,174 @@ This document describes all the REST API error codes that are used in WSO2 Ident
 </div>
+
+## Tenant Management Errors
+
+<div class="errorcode">
+<table>
+  <thead>
+    <th>Error Code</th>
+    <th>HTTP Status Code</th>
+    <th>Error Message</th>
+    <th>Possible Cause</th>
+  </thead>
+  <tbody>
+  <tr>
+    <td>TM-60000</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>Provided email is empty.</td>
+  </tr>
+  <tr>
+    <td>TM-60001</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>Wrong characters in the email.</td>
+  </tr>
+  <tr>
+    <td>TM-60002</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>Invalid email address is provided.</td>
+  </tr>
+  <tr>
+    <td>TM-60003</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>You can not use a registry reserved word as a tenant domain. Please choose a different one.</td>
+  </tr>
+  <tr>
+    <td>TM-60004</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>Provided domain name is empty.</td>
+  </tr>
+  <tr>
+    <td>TM-60005</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>You should have an extension to your domain.</td>
+  </tr>
+  <tr>
+    <td>TM-60006</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>Invalid domain. Domain should not start with '.'</td>
+  </tr>
+  <tr>
+    <td>TM-60007</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>The tenant domain %s contains one or more illegal characters. The valid characters are lowercase letters, numbers, '.', '-' and '_'.</td>
+  </tr>
+  <tr>
+    <td>TM-60008</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>User name : %s exists in the system. Please pick another user name for tenant Administrator.</td>
+  </tr>
+  <tr>
+    <td>TM-60009</td>
+    <td>400</td>
+    <td>Unable to add tenant.</td>
+    <td>A tenant with same domain %s already exist. Please use a different domain name.</td>
+  </tr>
+  <tr>
+    <td>TM-60010</td>
+    <td>400</td>
+    <td>Invalid Request</td>
+    <td>Limit should not be negative.</td>
+  </tr>
+  <tr>
+    <td>TM-60011</td>
+    <td>400</td>
+    <td>Invalid Request</td>
+    <td>Offset should not be negative.</td>
+  </tr>
+  <tr>
+    <td>TM-60012</td>
+    <td>400</td>
+    <td>Invalid Request</td>
+    <td>Required parameter owner is not specified.</td>
+  </tr>
+  <tr>
+    <td>TM-60013</td>
+    <td>400</td>
+    <td>Invalid Request</td>
+    <td>Required parameter %s is not specified.</td>
+  </tr>
+  <tr>
+    <td>TM-60014</td>
+    <td>400</td>
+    <td>Unable to retrieve tenant.</td>
+    <td>Tenant cannot be found for the provided id: %s.</td>
+  </tr>
+  <tr>
+    <td>TM-60015</td>
+    <td>400</td>
+    <td>Unable to retrieve tenant.</td>
+    <td>Tenant cannot be found for the provided domain: %s.</td>
+  </tr>
+  <tr>
+    <td>TM-60016</td>
+    <td>400</td>
+    <td>Error while deleting the tenant metadata.</td>
+    <td>Tenant deletion property Tenant.TenantDelete is not enabled in carbon.xml file./td>
```

Comment:
This description seems a bit unclear. Is `Tenant.TenantDelete` a property name? If so, shall we enclose it in <code>  tags?